actor/builtin/miner/miner_test.go

```diff
@@ -216,7 +215,7 @@ func TestMinerGetPower(t *testing.T) {
 
 		st, vms := core.CreateStorages(ctx, t)
 
-		minerAddr := createTestMinerWith(120, types.NewAttoFILFromFIL(240), t, st, vms, address.TestAddress,
+		minerAddr := createTestMinerWith(types.NewAttoFILFromFIL(240), t, st, vms, address.TestAddress,
```

Comment:
Why the choice of these numbers for the pledge (240 and 100 down below) and why 240 here (twice as much as before).  A comment saying that the number is enough to cover fees or not enough to cover fees or whatever the reason for that amount would be helpful.

Reply:
I didn't change anything here except removing an unused first parameter "pledge".